### ISSUE TITLE
You might want to indicate the version of AutoHotKey that this linter is for

### ISSUE THREAD

**Issue by LinuxOnTheDesktop** [eight04/ahk-linter#2]
I presume that the Linter is for version _1_ of AutoHotKey.

**Comment by eight04** [comment]
Yep. AHK 2 hadn't came out when I started the project. It aims to lint 1.1 scripts so authors can avoid those bad patterns e.g.
https://github.com/eight04/ahk-linter/blob/4426f209ee5a0cb4c42b8d09296e22390a9c3385/test/linter.js#L10-L11

However, after a while I moved most of my scripts to python or javascript so this project is incomplete.

I'll update the readme to reflect the current state.
